src/synchronizeContacts.js

```diff
@@ -5,15 +5,15 @@ const synchronizeContacts = (
 ) => {
   const {
     afterSave,
-    findRemoteDocument,
+    // findRemoteDocument,
     save,
     shouldSave
   } = synchronizationStrategy
   try {
     return Promise.all(
       localContacts.map(async localContact => {
-        const remoteContact = findRemoteDocument(localContact, remoteContacts)
-        if (shouldSave(remoteContact)) {
+        // const remoteContact = findRemoteDocument(localContact, remoteContacts)
+        if (shouldSave(localContact)) {
```

Comment:
why?

Reply:
because we now need to check if contact has a source account to know if we should save it